scripts/grep_pv

```diff
@@ -16,21 +16,24 @@ fi
 KEYWORD=$1
 
 if [[ -n $KEYWORD ]]; then
+    # shellcheck disable=SC2164
     cd /reg/d/iocData/
-    for i in ioc-*; do 
-	cd /reg/d/iocData/$i
+    for i in ioc-*; do
+	# shellcheck disable=SC2164
+	cd /reg/d/iocData/"$i"
```

Comment:
Perhaps _all_ the cd's in this file aren't actually necessary... we could easily just grep the full path. Do the cds in the loop make this easier to read?

Maybe I'm ambivalent about this in general. It's probably OK for these cds to fail because the next line checks for files.